axelar-chains-config/info/testnet.json

```diff
@@ -169,21 +170,22 @@
           "predeployCodehash": "0x73fc31262c4bad113c79439fd231281201c7c7d45b50328bd86bccf37684bf92"
         },
         "InterchainTokenService": {
-          "salt": "ITS v1.2.4",
+          "salt": "ITS v2.0.1",
           "deployer": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05",
-          "tokenManagerDeployer": "0x121b0e54Cd7ad2BBCb4c4C9275697978EBaF3653",
+          "tokenManagerDeployer": "0x3d36C02D7EE86F25565f319E71641CEFeD0eac81",
           "interchainToken": "0x7F9F70Da4af54671a6abAc58e705b5634cac8819",
-          "interchainTokenDeployer": "0x58667c5f134420Bf6904C7dD01fDDcB4Fea3a760",
-          "tokenManager": "0x81a0545091864617E7037171FdfcBbdCFE3aeb23",
-          "tokenHandler": "0x07715674F74c560200c7C95430673180812fCE73",
-          "implementation": "0x6d59D9360BDAe406614b7E61c53F43a03198A4ef",
+          "interchainTokenDeployer": "0xC605424cEF54f4Fe7FF7c6EC01a6D622d473E711",
+          "tokenManager": "0xc8e58E2BA863bFb86869227f8264d99de7738933",
+          "tokenHandler": "0x151198A9770107F01219Dfc5fc18e168DD529055",
+          "implementation": "0x0F55A4E925F41BB9B88A7D06ec11F14FD7ce6A6B",
           "address": "0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C",
```

Comment:
explorer has an issue verifying it as a proxy, see the warning here
https://sepolia.etherscan.io/address/0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C#readProxyContract

might need to contact explorer team if implementation verification is shown as successful by their API

Reply:
e.g. warning
>Implementation contract is now on [0x588c70b10e9eda627a2496b09edc69b79e2d72af](https://sepolia.etherscan.io/address/0x588c70b10e9eda627a2496b09edc69b79e2d72af#code) but NOT verified

Reply:
we have a bunch of verification tasks that we will handle after the fact. kava and some others are misbehaving.